backend/app/services/im/session_service.py

```
@@ -130,6 +130,35 @@ async def list_user_sessions(
+        if db is None:
+            return "wegent_chat"
+
+        from app.models.kind import Kind
+
+        channel = (
+            db.query(Kind)
+            .filter(
+                Kind.id == session.channel_id,
+                Kind.kind == "Messager",
+                Kind.user_id == 0,
+                Kind.is_active == True,
```

Ruff (0.15.18)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)